turn-lsp/org.jucmnav.turn.diagram/src/main/java/org/jucmnav/turn/diagram/TurnDiagramGenerator.java

```diff
@@ -10,10 +10,9 @@
 import io.typefox.sprotty.api.SModelRoot;
 import io.typefox.sprotty.server.xtext.IDiagramGenerator;
 
-//TODO: change class name once tested
-public class TurnDiagramOtherGenerator implements IDiagramGenerator {
+public class TurnDiagramGenerator implements IDiagramGenerator {
```

Comment:
Should this class be renamed to `TGRLDiagramGenerator`?
Same comment applies to `TurnDiagramModule.xtend`